Add team section (#64)
pythonindia/inpycon2021: content/team.md

---
title: Team
layout: page
---

# PyCon India 2021 Team

The conference wouldn't have concluded on a high note much less even started planning without the constant efforts of countless volunteers which were part of this year's workgroups. The following is a list of individuals that came together to work for 10+ months in order to make the online conference this year a great experience for everyone attending.

![A world map with names of pycon india volunteers](https://in.pycon.org/2021/images/team-world-map.png) 

- Abhishek Mishra
- Abhishek Upperwal
- Akshay Gaikwad
- Anirudh Menon
- Aravind Sekar
- Arun
- Bharat
- Bhavin Gandhi
- Chandan Kumar
- Darshna Das
- Divya Goswami
- Duranjai Malik
- Farhaan Bukhsh
- Gajendra Deshpande
- Garvit Kulshreshtha
- Girish Joshi

> - Hari priya

Reviewer: Not sure if this is correct name, her name on Zulip is Haripriya.

- Harsh Mittal
- Jatin Goel
- Kalyan Prasad
- Kumar Anirudha
- Laisha Wadhwa
- Megha

> - Minal Patil

Reviewer 2: Manaswini Das's name is missing. She helped me throughout stage 5.

- Minal Patil
- Nabarun Pal
- Nikhil Maan
- Pradhvan Bisht
- Praveen Patil
- Priyanshu Kumar
- Rahul B
- Rahul Kumaresan
- Rayan Das
- Ritesh Agrawal
- Rohan Samanta
- Sanchit Balchandani
- Sandeep Choudhary

> - Saptak S

Reviewer 3: Also Saptak Sengupta if we're using full name of everyone.

Reviewer: Saptak has asked to keep it Saptak S

> - Saptak Sengupta

Reviewer: https://github.com/pythonindia/inpycon2021/pull/64#discussion_r712787628

- Sayan Chowdhury
- Shashank Kumar
- Shivang Dubey

> - Subho

Reviewer 3: Can you please make it Subhendu Ghosh? It's his full name.

- Sukanya Mandal
- Tiyasha Banerjee
- Vaibhav Mishra
- Vipul Gupta

> - Vishal

Reviewer: ```suggestion
```

---
title: Team
layout: page
---

# PyCon India 2021 Team

The conference wouldn't have concluded on a high note much less even started planning without the constant efforts of countless volunteers which were part of this year's workgroups. The following is a list of individuals that came together to work for 10+ months in order to make the online conference this year a great experience for everyone attending.

- Abhishek Mishra
- Abhishek Upperwal
- Akshay Gaikwad
- Anirudh Menon
- Aravind Sekar
- Arun
- Bharat
- Bhavin Gandhi
- Chandan Kumar
- Darshna Das

> - Divya

Reviewer: ```suggestion
- Divya Goswami
```

- Duranjai Malik

> - Farhaan B

Reviewer: ```suggestion
- Farhaan Bukhsh
```

> - Gajendra

Reviewer: ```suggestion
- Gajendra Deshpande
```

- Garvit Kulshreshtha

> - Girish

Reviewer: ```suggestion
- Girish Joshi
```

- Hari priya
- Harsh Mittal
- Jatin Goel
- Kalyan Prasad
- Kumar Anirudha
- Laisha Wadhwa
- Megha

> - Minal

Reviewer: ```suggestion
- Minal Patil
```

- Nabarun Pal
- Nikhil Maan
- Pradhvan Bisht

> - Pravin Patil

Reviewer: ```suggestion
- Praveen Patil
```

- Priyanshu Kumar
- Rahul B
- Rahul Kumaresan
- Rayan Das

> - Ritesh

Reviewer: ```suggestion
- Ritesh Agrawal
```

- Rohan Samanta

> - Sanchit

Reviewer: ```suggestion
- Sanchit Balchandani
```

- Sandeep Choudhary

> - Saptak

Reviewer: ```suggestion
- Saptak Sengupta
```

Reviewer 4: `Saptak S` not `Saptak Sengupta`

> - Sayan

Reviewer: ```suggestion
- Sayan Chowdhury
```

- Shashank Kumar
- Shivang Dubey
- Subho
- Suchita Pallavi
- Sukanya Mandal
- Tiyasha Banerjee
- Vaibhav Mishra
- Vipul Gupta

> - Vishal

Reviewer: @realslimshanky did we have another Vishal than Vishal Vijayraghavan (who handled stage 4)?

Probably these two are just same.

Reviewer 5: @bhavin192 they are same.

> - Vishal VVR

Reviewer: ```suggestion
- Vishal Vijayraghavan
```

- Yashwanth Ravula

> This year, we had Chandan Kumar as the chair of the conference with Sayan Choudhary as the emertius chair. We also had 2 co-chairs, Kumar Aniruddha and Ritesh ?. If you too are interested in volunteering for next iteration of PyCon India, then [subscribe to the mailing list](https://mail.python.org/mailman/listinfo/inpycon) and wait for the call for volunteers for 2022 to start. 

Reviewer: ```suggestion
This year, we had Chandan Kumar as the chair of the conference with Sayan Chowdhury as the emertius chair. We also had 2 co-chairs, Kumar Anirudha and Ritesh Agrawal. If you too are interested in volunteering for next iteration of PyCon India, then [subscribe to the mailing list](https://mail.python.org/mailman/listinfo/inpycon) and wait for the call for volunteers for 2022 to start. 
```
